source/config/default_params/datasets/vdjdb_params.yaml

```diff
@@ -1,4 +1,5 @@
 is_repertoire: True
+path: ./
```

Comment:
if we set a default path where we will look for things, do we have this in the docs somewhere or do we have a proper error message? this is a reasonable default for galaxy, but not for CLI probably?

Reply:
it is in the docs of each of the import classes. it's maybe not super useful for CLI, but I don't think it hurts either?